### PR TITLE
fix(server): update all openshift urls

### PR DESCRIPTION
**Pull request by dgutride**

### CODE REVIEW

File: vars/buildSnapshotFabric8UI.groovy [hunk 1]
```diff
@@ -29,7 +29,7 @@ def call(body) {
         export FABRIC8_SSO_API_URL="https://sso.prod-preview.openshift.io/"
         export FABRIC8_AUTH_API_URL="https://auth.prod-preview.openshift.io/api/"
         
-        export OPENSHIFT_CONSOLE_URL="https://console.starter-us-east-2.openshift.com/console/"
```
Comment (joshuawilson) [inline vars/buildSnapshotFabric8UI.groovy]: how do you know this should be changed and to this value?

File: vars/deployOpenShiftSnapshot.groovy [hunk 2]
```diff
@@ -60,8 +60,8 @@ def call(body) {
             export FABRIC8_SSO_API_URL="https://sso.prod-preview.openshift.io/"
             export FABRIC8_AUTH_API_URL="https://auth.prod-preview.openshift.io/api/"
 
-            export OPENSHIFT_CONSOLE_URL="https://api.free-stg.openshift.com/console/"
```
Comment (joshuawilson) [inline vars/deployOpenShiftSnapshot.groovy]: The env var looks like this change makes sense but if it has been working do we need this change?

Either it isn't used OR it already works OR the change in to the proxy requires this for some reason.

Comment (dgutride) [reply vars/deployOpenShiftSnapshot.groovy]: That value is incorrect now - it does not work.  Setting them all consistently will help with removal in our next PR.

File: vars/buildSnapshotFabric8UI.groovy (hunk 1 above)
Comment (dgutride) [reply vars/buildSnapshotFabric8UI.groovy]: That value is incorrect - changing to the correct free-stg value will at least help us identify where it's broken in the deploy so it's consistent.

**Pull request merged**